saleor/graphql/account/tests/mutations/test_token_create.py

```diff
@@ -109,3 +109,38 @@ def test_create_token_invalid_email(api_client, customer_user):
     response_error = content["data"]["tokenCreate"]["errors"][0]
     assert response_error["code"] == expected_error_code
     assert response_error["field"] == "email"
+
```

Comment:
Do we have a test for scenario when user is active and has last login? If not then would be good to add it